dspace/cli/senior_thesis_community.rb

```diff
@@ -0,0 +1,354 @@
+module DSpace
+  module CLI
+    class SeniorThesisCollection
+      attr_reader :obj
+
+      def self.kernel
+        ::DSpace
+      end
+
+      def initialize(obj)
```

Comment:
There are methods such as these which shouldn't be needed - I will try and push these into a base Class before opening this PR for review.

Reply:
I should specify here that `DCommunity` and `DItem` did not permit me to use base classes and override the constructor.